common/src/test/java/com/microsoft/identity/common/crypto/AndroidAuthSdkStorageEncryptionManagerTest.java

```diff
@@ -121,6 +121,22 @@ public void testGetDecryptionKey_ForDataEncryptedWithPreDefinedKey() {
         }
     }
 
+    @Test
+    public void testGetDecryptionKey_ForDataEncryptedWithPreDefinedKey_PredefinedKeyNotProvided_returns_null_When_shouldIgnorePredefinedKeyLoaderNotFoundError_is_true() {
```

Comment:
Nit: Very long method names are unreadable. Is it possible to capture the context in a comment and keep the method name short and meaningful?

Reply:
I have shortened it.
This is the name of the test and makes it easier to read the Test failure reports to see exactly what test failed
That is the reason I prefer to keep the name in this format "<what_Is_being_Tested>_<With_Preconditions_or_Inputs>_<expected_behavior>"